docs/resources/job.md

```diff
@@ -3,63 +3,45 @@
 page_title: "qovery_job Resource - terraform-provider-qovery"
 subcategory: ""
 description: |-
-  Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
+  Provides a Qovery job resource. This can be used to create and manage Qovery job registry. Cron job example https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-cron-job Lifecycle job example https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-lifecycle-job
```

Comment:
what does Qovery job registry means?

Reply:
instead we should explain that this is used to deploy the lifecycle jobs or cronjobs as explained in the doc https://hub.qovery.com/docs/using-qovery/configuration/cronjob/ https://hub.qovery.com/docs/using-qovery/configuration/lifecycle-job/

Reply:
> what does Qovery job registry means? 

It's a copy paste I guess, need to find the root go file generating this one.

Reply:
Everything should be updated in this file https://github.com/Qovery/terraform-provider-qovery/blob/main/qovery/resource_job.go#L56

Then, you need to run `task docs`